workers/unity/Assets/Fps/Scripts/SetupLogic/OneTimeInitialisation.cs

```diff
@@ -18,8 +16,6 @@ private static void Init()
             }
 
             initialized = true;
-            WorldsInitializationHelper.SetupInjectionHooks();
-            PlayerLoopManager.RegisterDomainUnload(WorldsInitializationHelper.DomainUnloadShutdown, 1000);
```

Comment:
that not needed anymore? (the domainunloadshutdown bit)

Reply:
You're right, deleted a line too many. Putting this back.